app/slimming/slimming.js

```diff
@@ -6,6 +6,29 @@ app.controller('GOSlimCtrl', function($scope, $location, $q,
   $scope.selection = {};
   $scope.deSelectedItems = [];
 
+
+
+  function sticky_relocate() {
```

Comment:
Could you please not copy/paste code found on the internet (https://jsfiddle.net/livibetter/HV9HM/) without making sure it fits the existing code and standards (camel case instead of _, ). See other comment below.